js/components/portal-dashboard/response-details/response-details.tsx

```diff
@@ -124,29 +141,64 @@ export class ResponseDetails extends React.PureComponent<IProps, IState> {
             }
             </div>
           </div>
+          {viewMode === "FeedbackReport" &&
```

Comment:
I'd probably add a space to be consistent with other cases in this file:
`{ viewMode === "FeedbackReport" &&`